.circleci/config.yml

```diff
@@ -63,14 +66,18 @@ jobs:
   test-client:
     working_directory: ~/metashare
     docker:
-      - image: circleci/python:3.7.3-node
+      - image: wlonk/oddbird
+    environment:
+      VIRTUAL_ENV: ./venv
     steps:
       - checkout
       - attach_workspace:
           at: .
       - run:
           name: Test JS
-          command: yarn test:coverage
+          command: |
+            . venv/bin/activate
```

Comment:
Why does the venv need to be active to run `yarn`?

Reply:
It doesn't, here. This was just a side-effect of thrashing.